tests/test_experimental/test_prediction_intervals/test_base.py

```diff
@@ -0,0 +1,180 @@
+    intervals_pipeline_pred = intervals_pipeline.forecast(prediction_interval=False)
+
+    pd.testing.assert_frame_equal(intervals_pipeline_pred.df, pipeline_pred.df)
+
+
+@pytest.mark.parametrize(
+    "expected_columns",
+    ({"target_lower", "target_upper"},),
+)
+@pytest.mark.parametrize("pipeline_name", ("naive_pipeline", "naive_pipeline_with_transforms"))
+def test_forecast_intervals_exists(example_tsds, pipeline_name, expected_columns, request):
+    pipeline = request.getfixturevalue(pipeline_name)
+
+    intervals_pipeline = DummyPredictionIntervals(pipeline=pipeline)
+    intervals_pipeline.fit(ts=example_tsds)
+
+    intervals_pipeline_pred = intervals_pipeline.forecast(prediction_interval=True)
+    columns = intervals_pipeline_pred.df.columns.get_level_values("feature")
+
+    assert len(expected_columns - set(columns)) == 0
+
+
+@pytest.mark.parametrize(
+    "expected_columns",
+    ({"target", "target_lower", "target_upper"},),
+)
+@pytest.mark.parametrize(
+    "pipeline",
+    (
+        Pipeline(model=NaiveModel(), horizon=1),
+        AutoRegressivePipeline(model=NaiveModel(), horizon=1),
+        HierarchicalPipeline(
+            model=NaiveModel(),
+            horizon=1,
+            reconciliator=BottomUpReconciliator(target_level="market", source_level="product"),
+        ),
+    ),
+)
+def test_pipelines_forecast_intervals(product_level_constant_hierarchical_ts, pipeline, expected_columns):
+    run_base_pipeline_compat_check(
+        ts=product_level_constant_hierarchical_ts, pipeline=pipeline, expected_columns=expected_columns
+    )
+
+
+@pytest.mark.parametrize(
+    "expected_columns",
+    ({"target", "target_lower", "target_upper"},),
+)
+@pytest.mark.parametrize(
+    "ensemble",
+    (
+        DirectEnsemble(pipelines=[get_naive_pipeline(horizon=1), get_naive_pipeline_with_transforms(horizon=2)]),
+        VotingEnsemble(pipelines=[get_naive_pipeline(horizon=1), get_naive_pipeline_with_transforms(horizon=1)]),
+        StackingEnsemble(pipelines=[get_naive_pipeline(horizon=1), get_naive_pipeline_with_transforms(horizon=1)]),
+    ),
+)
+def test_ensembles_forecast_intervals(example_tsds, ensemble, expected_columns):
+    run_base_pipeline_compat_check(ts=example_tsds, pipeline=ensemble, expected_columns=expected_columns)
+
+
+@pytest.mark.parametrize(
+    "pipeline",
+    (
+        get_naive_pipeline(horizon=1),
+        get_naive_pipeline_with_transforms(horizon=2),
+        Pipeline(model=CatBoostPerSegmentModel()),
+        AutoRegressivePipeline(model=CatBoostPerSegmentModel(), horizon=1),
+        HierarchicalPipeline(
+            model=NaiveModel(),
+            horizon=1,
+            reconciliator=BottomUpReconciliator(target_level="market", source_level="product"),
+        ),
+    ),
+)
+def test_default_params_to_tune(pipeline):
+    intervals_pipeline = DummyPredictionIntervals(pipeline=pipeline)
+    assert intervals_pipeline.params_to_tune() == pipeline.params_to_tune()
```

Comment:
It shouldn't work like this. Because the `pipeline.` prefix should be added to each param in case of `intervals_pipeline`.

Reply:
What are we going to do with this check? Are we going to fix it in some other task related to `params_to_tune`?